app/helpers/carts_helper.rb

```diff
@@ -0,0 +1,7 @@
+module CartsHelper
+  def current_cart_number
+    count = 0
+    session[:cart]&.each{|_, v| count += v.to_i}
+    count
```

Comment:
tìm hiểu `inject in ruby` đi cu 

ngoài ra, hash nếu em chỉ lấy value thì chỉ cần `.values` thôi hì.